aten/src/ATen/native/cpu/UnaryOpsKernel.cpp

```diff
@@ -395,6 +396,15 @@ static void clamp_min_kernel(TensorIterator& iter, Scalar min_scalar) {
   });
 }
 
+static void kaiser_window_kernel(TensorIterator& iter, int64_t window_length, double beta){
+  AT_DISPATCH_FLOATING_TYPES_AND(kBFloat16, iter.dtype(), "kaiser_window_cpu", [&](){
+    auto alpha = static_cast<scalar_t>((window_length - 1) / 2.0);
```

Comment:
This looks correct now. As a nit you could do `const scalar_t alpha = ...` to be more explicit.